src/typeGuards.ts

```diff
@@ -30,7 +28,7 @@ export function checkSchema(schemaKey: string, obj: object): SchemaResult {
     if (_cachedAjv.errors?.length) {
       throw new Error("Object matches schema, but errors are listed: ${");
```

Comment:
Unrelated, but what is the curious `${` at the end of this error?

Reply:
A mistake. I had intended to make this list the errors, but apparently forgot to finish my sentence.